llm_studio/src/utils/modeling_utils.py

```diff
@@ -504,7 +499,9 @@ def create_nlp_backbone(cfg, model_class=AutoModel, kwargs={}) -> Any:
         backbone.resize_token_embeddings(cfg.tokenizer._vocab_length)
 
     if cfg.training.lora:
-        backbone = prepare_model_for_lora_training(backbone, layer_norm_names=[])
+        backbone = prepare_model_for_kbit_training(
+            backbone, use_gradient_checkpointing=cfg.architecture.gradient_checkpointing
```

Comment:
We are doing the gradient checkpointing code already outside. So this needs to be set to False.

See the old function `prepare_model_for_lora_training`.

Reply:
It shouldn't hurt, right?

If so, I would leave it like this to avoid confusion then

Reply:
It is redundant and might lead to issues. I would just set it to False there.